README.md

```diff
@@ -48,7 +48,14 @@ _Tip: Consider using the faster and smaller ES6 build if targetting a modern env
 | 4.\*         | Yes (LTS)          | Any ES5 compliant browser                                                                                                                                   | `mobx4-master`   |
 | 1-3.\*       | No                 | Any ES5 compliant browser                                                                                                                                   | No active branch |
 
--   MobX >=5 runs on any browser with [ES6 proxy support](https://kangax.github.io/compat-table/es6/#test-Proxy). It will throw an error on startup on older environments such as IE11, Node.js <6 or React Native Android on old JavaScriptCore [how-to-upgrade](https://github.com/react-community/jsc-android-buildscripts#how-to-use-it-with-my-react-native-app).
+-   MobX >=5 runs on any browser with [ES6 proxy support](https://kangax.github.io/compat-table/es6/#test-Proxy). In practice this means:
+    - no Internet Explorer (Edge is fine)
+    - Node.js >= 6  
+React Native:
+    - iOS >= 10
+    - Android from RN 0.59 (or with manual JavaScript core [upgrade](https://github.com/react-community/jsc-android-buildscripts#how-to-use-it-with-my-react-native-app))
+    - Hermes runtime is [not supported](https://github.com/facebook/hermes/issues/28)  
+All modern browsers are supported.
```

Comment:
```suggestion
    - All modern browsers are supported.
```

Reply:
Applied

Reply:
Sorry was unavailable. Was this expected change that you removed the sub-items?

*Browser support* section looks like
![2019-07-29-125915_892x141_scrot](https://user-images.githubusercontent.com/2343101/62039725-e39d6d00-b200-11e9-8822-b6ba90d6a4f9.png)

In my branch 
![2019-07-29-130248_764x196_scrot](https://user-images.githubusercontent.com/2343101/62039871-2e1ee980-b201-11e9-9ee6-89394c5182c1.png)

Reply:
Well, it's mostly the same, just restructured differently. All the information is there.